src/views/view-final-confirmation.njk

```diff
@@ -22,12 +22,12 @@
 
       <h2 class="govuk-heading-m">What to do next</h2>
 
-      <p class="govuk-body"><a href="{{ Paths.CERTIFICATE_DOWNLOAD_URI }}" target="_blank" class="govuk-link">Download the signed copy of the application (opens in a new tab)</a>.</p>
+      <p class="govuk-body"><a href="#" target="_blank" onclick="javascript:_paq.push(['trackGoal', '{{ piwik.PIWIK_CONFIRMATION_PAGE_PDF_GOAL_ID }}');" class="govuk-link">Download the signed copy of the application (PDF 238Kb) (opens in a new tab)</a>.</p>
```

Comment:
Why have you changed the href? This hyperlink currently does nothing. Also why are you adding the (PDF 228kb) text? This story is just about piwik, there's nothing in the ACs about updating content. Was this called out at 3 amigos or something?

Reply:
I have no idea why this is appearing, editing it out